chipsec/file.py

```diff
@@ -75,6 +74,14 @@ def write_file(filename: str, buffer: Any, append: bool = False) -> bool:
     return True
 
 
+def write_unique_file(file_buffer: Any, file_name: str = '', file_extension: str = '') -> bool:
+    """Writes file with the name <file_name>_<year><month><day>-<hour><minute><second>.<file_extension>"""
+    file_str = f'{file_name}_' if file_name else ''
+    file_ext = f'.{file_extension}' if file_extension else ''
```

Comment:
If we're planning on having something consuming the file, does it make sense to return the filename in that case as well as the pass/fail bool?

Reply:
No, if the write fails, return an empty string.  Why would you need an additional bool?

Reply:
no, my thought was, if you want something to consume the file, might as well return the file name like: 
```python
def write_unique_file(file_buffer: Any, file_name: str = '', file_extension: str = '') -> (bool, str):
    #...
    return (write_file(file_name_str, file_buffer), file_name_str)
```
That way you can either print the file name or open it if you want/need to consume it. Otherwise you have no idea what the filename is.